variables.tf

```
@@ -0,0 +1,29 @@
+variable "region" {
+  description = "Region"
+  type        = string
+  default     = "us-west-2"
+}
+
+variable "name" {
+  description = "Name of the VPC and EKS Cluster"
+  type        = string
+  default     = "aerospike-on-eks"
+}
+
+variable "eks_cluster_version" {
+  description = "EKS Cluster version"
+  type        = string
+  default     = "1.30"
+}
+
+variable "aerospike_operator_version" {
+  description = "Aerospike Kubernetes Operator version"
+  type        = string
+  default     = "3.3.1"
```

```suggestion
  default     = "3.4.0"
```

use the latest operator version

---

done